lisa/sut_orchestrator/baremetal/cluster/rackmanager.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass
+from typing import Any, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import RemoteNode, schema
+from lisa.sut_orchestrator.baremetal.cluster.cluster import Cluster, ClusterSchema
+from lisa.util import fields_to_dict
+from lisa.util.logger import get_logger
+from lisa.util.shell import try_connect
+
+
+@dataclass_json()
+@dataclass
+class RackManagerSchema(ClusterSchema):
+    address: str = ""
+    username: str = ""
+    password: str = ""
+    port: int = 22
+
+
+class RackManager(Cluster):
+    def __init__(self, runbook: RackManagerSchema) -> None:
+        super().__init__(runbook)
+        self.rm_runbook: RackManagerSchema = self.runbook
+        self._log = get_logger("rackmanager", self.__class__.__name__)
+
+    @classmethod
+    def type_name(cls) -> str:
+        return "rackmanager"
+
+    @classmethod
+    def type_schema(cls) -> Type[schema.TypedSchema]:
+        return RackManagerSchema
+
+    def deploy(self) -> Any:
+        self._log.debug("deploy client through rackmanager")
+        connection_info = schema.ConnectionInfo(
+            address=self.rm_runbook.address,
+            port=self.rm_runbook.port,
+            username=self.rm_runbook.username,
+            password=self.rm_runbook.password,
+        )
+        rack_manager = RemoteNode(
+            runbook=schema.Node(name="rackmanager"),
+            index=-1,
+            logger_name="rackmanager",
+            parent_logger=get_logger("baremetal-platform"),
+        )
+        rack_manager.set_connection_info(
+            public_port=self.rm_runbook.port,
+            **fields_to_dict(
+                connection_info, ["address", "port", "username", "password"]
+            ),
+        )
+        try_connect(connection_info)
+        assert self.rm_runbook.client
+        rack_manager.execute(
+            f"set system reset -i {self.rm_runbook.client.management_port}"
```

Comment:
the client should be passed in, instead of hardcode here. There maybe multiple clients in a rack manager cluster.